velox/type/TimestampConversion.cpp

```diff
@@ -733,4 +734,13 @@ Timestamp fromTimestampString(const char* str, size_t len) {
   return timestamp;
 }
 
+std::tuple<std::string, std::string> splitTimestampWithTimezoneString(std::string& tsTzStr)
```

Comment:
Should be `const &` parameter.

Reply:
Yes, this wasn't yet ready for review, PR not updated.

Reply:
Yes, this wasn't yet ready for review, PR not updated.